README.md

```diff
@@ -3,12 +3,9 @@
 
 The Coinbase Go SDK enables the simple integration of crypto into your app. By calling Coinbase's Platform APIs, the SDK allows you to provision crypto wallets, send crypto into/out of those wallets, track wallet balances, and trade crypto from one asset into another.
 
-The SDK currently supports Customer-custodied Wallets on the Base Sepolia test network.
+The SDK supports various verbs on Customer-custodied Wallets across multiple networks, as documented [here](https://docs.cdp.coinbase.com/cdp-sdk/docs/networks).
```

Comment:
nit: Is Customer-custodied Wallets a widely used term in CDP ? Can we call out the fact that this currently only works on External Addresses ?

Reply:
We call it: `End-User Wallets` everywhere in the docs so I update to that!